requirements.txt

```diff
@@ -1,13 +1,15 @@
-numpy==1.15.*
-matplotlib==3.0.*
+.
```

Comment:
I think this is a really bad idea. I would never expect that `pip install -r requirements.txt` installs the package it self. That breaks a fundamental contract with the user. More over this is unwanted in cases were I want to install all the dependencies as regular packages but install qcodes as editable. In the best case this results in a redundant install of qcodes.